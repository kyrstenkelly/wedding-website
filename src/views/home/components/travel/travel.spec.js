import React from 'react';
import { shallow } from 'enzyme';

import Travel from './travel';

describe('<Travel />', () => {
  it('matches the snapshot', () => {
    expect(shallow(<Travel />)).toMatchSnapshot();
  });
});
