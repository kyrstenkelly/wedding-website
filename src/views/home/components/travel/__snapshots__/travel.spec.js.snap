// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Travel /> matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Travel />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="section"
>
          <div
                    className="title-main"
          >
                    Airports
          </div>
          <p>
                    The closest airport to the venue is 
                    <Link
                              href="https://www.google.com/maps/dir/Norman+Y.+Mineta+San+Jose+International+Airport+(SJC),+1701+Airport+Blvd,+San+Jose,+CA+95110/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.298684,-121.9812996,12z/data=!4m8!4m7!1m2!1m1!1s0x808fcbc3fab3c59b:0xbcfa443f6df67e3e!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                              label="San Jose Airport (SJC)"
                    />
                    ,
                    <br />
                    and 
                    <Link
                              href="https://www.google.com/maps/dir/San+Francisco+Airport+(SFO),+San+Francisco,+CA/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.4355891,-122.2161904,11z/data=!4m8!4m7!1m2!1m1!1s0x808f778c55555555:0xa4f25c571acded3f!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                              label="San Fransisco Airport (SFO)"
                    />
                     is the next closest.
          </p>
</div>,
        <div
          className="section"
>
          <div
                    className="title-main"
          >
                    Hotels
          </div>
          <p>
                    We will arrange a block of hotels rooms somewhere nearby. Check back soon for more details.
          </p>
</div>,
      ],
      "className": "travel",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="title-main"
>
              Airports
</div>,
            <p>
              The closest airport to the venue is 
              <Link
                            href="https://www.google.com/maps/dir/Norman+Y.+Mineta+San+Jose+International+Airport+(SJC),+1701+Airport+Blvd,+San+Jose,+CA+95110/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.298684,-121.9812996,12z/data=!4m8!4m7!1m2!1m1!1s0x808fcbc3fab3c59b:0xbcfa443f6df67e3e!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                            label="San Jose Airport (SJC)"
              />
              ,
              <br />
              and 
              <Link
                            href="https://www.google.com/maps/dir/San+Francisco+Airport+(SFO),+San+Francisco,+CA/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.4355891,-122.2161904,11z/data=!4m8!4m7!1m2!1m1!1s0x808f778c55555555:0xa4f25c571acded3f!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                            label="San Fransisco Airport (SFO)"
              />
               is the next closest.
</p>,
          ],
          "className": "section",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Airports",
              "className": "title-main",
            },
            "ref": null,
            "rendered": "Airports",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "The closest airport to the venue is ",
                <Link
                  href="https://www.google.com/maps/dir/Norman+Y.+Mineta+San+Jose+International+Airport+(SJC),+1701+Airport+Blvd,+San+Jose,+CA+95110/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.298684,-121.9812996,12z/data=!4m8!4m7!1m2!1m1!1s0x808fcbc3fab3c59b:0xbcfa443f6df67e3e!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                  label="San Jose Airport (SJC)"
/>,
                ",",
                <br />,
                "and ",
                <Link
                  href="https://www.google.com/maps/dir/San+Francisco+Airport+(SFO),+San+Francisco,+CA/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.4355891,-122.2161904,11z/data=!4m8!4m7!1m2!1m1!1s0x808f778c55555555:0xa4f25c571acded3f!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                  label="San Fransisco Airport (SFO)"
/>,
                " is the next closest.",
              ],
            },
            "ref": null,
            "rendered": Array [
              "The closest airport to the venue is ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "href": "https://www.google.com/maps/dir/Norman+Y.+Mineta+San+Jose+International+Airport+(SJC),+1701+Airport+Blvd,+San+Jose,+CA+95110/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.298684,-121.9812996,12z/data=!4m8!4m7!1m2!1m1!1s0x808fcbc3fab3c59b:0xbcfa443f6df67e3e!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0",
                  "label": "San Jose Airport (SJC)",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              ",",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              "and ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "href": "https://www.google.com/maps/dir/San+Francisco+Airport+(SFO),+San+Francisco,+CA/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.4355891,-122.2161904,11z/data=!4m8!4m7!1m2!1m1!1s0x808f778c55555555:0xa4f25c571acded3f!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0",
                  "label": "San Fransisco Airport (SFO)",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              " is the next closest.",
            ],
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="title-main"
>
              Hotels
</div>,
            <p>
              We will arrange a block of hotels rooms somewhere nearby. Check back soon for more details.
</p>,
          ],
          "className": "section",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Hotels",
              "className": "title-main",
            },
            "ref": null,
            "rendered": "Hotels",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "We will arrange a block of hotels rooms somewhere nearby. Check back soon for more details.",
            },
            "ref": null,
            "rendered": "We will arrange a block of hotels rooms somewhere nearby. Check back soon for more details.",
            "type": "p",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="section"
>
            <div
                        className="title-main"
            >
                        Airports
            </div>
            <p>
                        The closest airport to the venue is 
                        <Link
                                    href="https://www.google.com/maps/dir/Norman+Y.+Mineta+San+Jose+International+Airport+(SJC),+1701+Airport+Blvd,+San+Jose,+CA+95110/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.298684,-121.9812996,12z/data=!4m8!4m7!1m2!1m1!1s0x808fcbc3fab3c59b:0xbcfa443f6df67e3e!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                                    label="San Jose Airport (SJC)"
                        />
                        ,
                        <br />
                        and 
                        <Link
                                    href="https://www.google.com/maps/dir/San+Francisco+Airport+(SFO),+San+Francisco,+CA/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.4355891,-122.2161904,11z/data=!4m8!4m7!1m2!1m1!1s0x808f778c55555555:0xa4f25c571acded3f!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                                    label="San Fransisco Airport (SFO)"
                        />
                         is the next closest.
            </p>
</div>,
          <div
            className="section"
>
            <div
                        className="title-main"
            >
                        Hotels
            </div>
            <p>
                        We will arrange a block of hotels rooms somewhere nearby. Check back soon for more details.
            </p>
</div>,
        ],
        "className": "travel",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="title-main"
>
                Airports
</div>,
              <p>
                The closest airport to the venue is 
                <Link
                                href="https://www.google.com/maps/dir/Norman+Y.+Mineta+San+Jose+International+Airport+(SJC),+1701+Airport+Blvd,+San+Jose,+CA+95110/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.298684,-121.9812996,12z/data=!4m8!4m7!1m2!1m1!1s0x808fcbc3fab3c59b:0xbcfa443f6df67e3e!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                                label="San Jose Airport (SJC)"
                />
                ,
                <br />
                and 
                <Link
                                href="https://www.google.com/maps/dir/San+Francisco+Airport+(SFO),+San+Francisco,+CA/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.4355891,-122.2161904,11z/data=!4m8!4m7!1m2!1m1!1s0x808f778c55555555:0xa4f25c571acded3f!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                                label="San Fransisco Airport (SFO)"
                />
                 is the next closest.
</p>,
            ],
            "className": "section",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Airports",
                "className": "title-main",
              },
              "ref": null,
              "rendered": "Airports",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "The closest airport to the venue is ",
                  <Link
                    href="https://www.google.com/maps/dir/Norman+Y.+Mineta+San+Jose+International+Airport+(SJC),+1701+Airport+Blvd,+San+Jose,+CA+95110/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.298684,-121.9812996,12z/data=!4m8!4m7!1m2!1m1!1s0x808fcbc3fab3c59b:0xbcfa443f6df67e3e!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                    label="San Jose Airport (SJC)"
/>,
                  ",",
                  <br />,
                  "and ",
                  <Link
                    href="https://www.google.com/maps/dir/San+Francisco+Airport+(SFO),+San+Francisco,+CA/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.4355891,-122.2161904,11z/data=!4m8!4m7!1m2!1m1!1s0x808f778c55555555:0xa4f25c571acded3f!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0"
                    label="San Fransisco Airport (SFO)"
/>,
                  " is the next closest.",
                ],
              },
              "ref": null,
              "rendered": Array [
                "The closest airport to the venue is ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "href": "https://www.google.com/maps/dir/Norman+Y.+Mineta+San+Jose+International+Airport+(SJC),+1701+Airport+Blvd,+San+Jose,+CA+95110/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.298684,-121.9812996,12z/data=!4m8!4m7!1m2!1m1!1s0x808fcbc3fab3c59b:0xbcfa443f6df67e3e!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0",
                    "label": "San Jose Airport (SJC)",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                ",",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": "br",
                },
                "and ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "href": "https://www.google.com/maps/dir/San+Francisco+Airport+(SFO),+San+Francisco,+CA/Saratoga+Springs+Events+%26+Weddings,+Big+Basin+Way,+Saratoga,+CA/@37.4355891,-122.2161904,11z/data=!4m8!4m7!1m2!1m1!1s0x808f778c55555555:0xa4f25c571acded3f!1m2!1m1!1s0x808e4b43b6696425:0x5eb7cbd719ad3eea!3e0",
                    "label": "San Fransisco Airport (SFO)",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                " is the next closest.",
              ],
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="title-main"
>
                Hotels
</div>,
              <p>
                We will arrange a block of hotels rooms somewhere nearby. Check back soon for more details.
</p>,
            ],
            "className": "section",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Hotels",
                "className": "title-main",
              },
              "ref": null,
              "rendered": "Hotels",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "We will arrange a block of hotels rooms somewhere nearby. Check back soon for more details.",
              },
              "ref": null,
              "rendered": "We will arrange a block of hotels rooms somewhere nearby. Check back soon for more details.",
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": false,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": true,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
